products/bluebell/cli/src/main.rs

```diff
@@ -27,8 +27,38 @@ use bluebell::intermediate_representation::pass_manager::PassManager;
 use bluebell::parser::lexer;
 use bluebell::parser::lexer::Lexer;
 use bluebell::parser::{parser, ParserError};
-
 use evm_assembly::types::EvmTypeValue;
+use log::{Log, Metadata, Record};
+
+struct CaptureLogger {}
```

Comment:
How does this differ from the default logger in `log`?

Reply:
Oh it doesn't come with one - Does it make sense to use one of the implementations here https://docs.rs/log/latest/log/#available-logging-implementations ?

Reply:
That might make sense. In fact, I did start with the default one, but then changed it. The reason I changed it was because I am slightly "misusing" the logger to create a similar experience in the browser as in the terminal (see `playground/main.rs`) to allow us to implement runtime functionality that prints to stdout. This probably should be revised at some point, but for now it serves to show that this approach works and provide some interim solution until I come up with something better.